docs/src/development/logs.md

```diff
@@ -92,6 +92,7 @@ file=none
 
 {{< /codetabs >}}
 
+UTC is the default timezone for all logs.
```

Comment:
I feel like it might get lost here. Doesn't it also apply to the activity logs? At least as a separate paragraph might be clearer, since it doesn't relate to the rest.